cluster-controller/src/main/java/io/strimzi/controller/cluster/ClusterController.java

```diff
@@ -37,6 +37,7 @@
     public static final String STRIMZI_TYPE_LABEL = STRIMZI_DOMAIN + "/type";
     public static final String STRIMZI_CLUSTER_LABEL = STRIMZI_DOMAIN + "/cluster";
     public static final String STRIMZI_NAME_LABEL = STRIMZI_DOMAIN + "/name";
+    public static final String STRMIZI_CONTROLLER_SERVICE_ACCOUNT = "strimzi-cluster-controller";
```

Comment:
Typo: STRMIZI instead of STRIMZI.

Reply:
On a second look ... it should probably also say `STRIMZI_CLUSTER_CONTROLLER_SERVICE_ACCOUNT` to make it clear this is cluster controller account.